src/functionalTest/resources/responses/response-caseworker-offline-document-verified-d84-about-to-submit.json

```diff
@@ -20,28 +20,6 @@
     },
     "coDateD84FormScanned": "2022-01-01T12:12",
     "documentsGenerated": [{
-      "id": "${json-unit.any-string}",
-      "value": {
-        "documentLink": {
-          "document_url": "${json-unit.any-string}",
-          "document_filename": "${json-unit.any-string}",
-          "document_binary_url": "${json-unit.any-string}"
-        },
-        "documentFileName": "${json-unit.any-string}",
-        "documentType": "appliedForCoLetter"
-      }
-    }, {
-      "id": "${json-unit.any-string}",
-      "value": {
-        "documentLink": {
-          "document_url": "${json-unit.any-string}",
-          "document_filename": "${json-unit.any-string}",
-          "document_binary_url": "${json-unit.any-string}"
-        },
-        "documentFileName": "${json-unit.any-string}",
-        "documentType": "appliedForCoLetter"
-      }
-    }, {
```

Comment:
was this duplicate ?

Reply:
@nitinprabhu These documents are generated during notification. That is why i removed it from the response when i moved the notification related code to submitted callback